src/app/components/MobileFooterNavigation/index.tsx

```diff
@@ -23,15 +25,27 @@ const StyledMobileFooterNavigation = styled.nav`
   right: 0;
   height: ${mobileFooterNavigationHeight};
   display: flex;
-  align-items: center;
-  justify-content: space-evenly;
+  align-items: stretch;
+  justify-content: space-between;
   box-shadow: ${({ theme }) =>
     `0px 0px ${theme.global?.borderSize?.xsmall} ${normalizeColor('background-front-border', theme)}`};
   border-top: ${({ theme }) =>
     `solid ${theme.global?.borderSize?.xsmall} ${normalizeColor('background-contrast', theme)}`};
   flex-direction: row;
 `
 
+const StyledNavLink = styled(NavLink)`
+  // Make items mostly-equal width
+  flex-grow: 1;
+  flex-basis: 0;
+  &:first-of-type {
+    flex-grow: 1.1;
+  }
```

Comment:
Not a fan of this, would rather see a rename from "ROSE Wallet" -> "Wallet" to avoid this.

Reply:
I think we are forced to keep "ROSE Wallet"